snmp/README.md

```diff
@@ -236,40 +127,204 @@ instances:
+|------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------|
+| `snmp.entSensorValue`              | The most recent measurement seen by the sensor.                                                                                                                                                                        | `sensor_id`, `sensor_type` |
+| `snmp.cefcFRUPowerAdminStatus`     | Administratively desired FRU power state.                                                                                                                                                                              | `fru`                      |
+| `snmp.cefcFRUPowerOperStatus`      | Operational FRU power state.                                                                                                                                                                                           | `fru`                      |
+| `snmp.cefcFRUCurrent`              | Current supplied by the FRU (positive values) or current required to operate the FRU (negative values).                                                                                                                | `fru`                      |
+| `snmp.cpmCPUTotalMonIntervalValue` | The overall CPU busy percentage in the last cpmCPUMonInterval period.                                                                                                                                                  | `cpu`                      |
+| `snmp.cpmCPUMemoryUsed`            | The overall CPU wide system memory currently in use.                                                                                                                                                       | `cpu`                      |
+| `snmp.cpmCPUMemoryFree`            | The overall CPU wide system memory currently free.                                                                                                                                                            | `cpu`                      |
+| `snmp.cieIfLastInTime`             | The elapsed time in milliseconds since the last protocol input packet was received.                                                                                                                                        | `interface`                |
+| `snmp.cieIfLastOutTime`            | The elapsed time in milliseconds since the last protocol  output packet was transmitted.                                                                                                                                   | `interface`                |
```

Comment:
```suggestion
| `snmp.cieIfLastOutTime`            | The elapsed time in milliseconds since the last protocol output packet was transmitted.                                                                                                                                   | `interface`                |
```